source/documentation/administration_guide/chap-Renewing_certificates.adoc

```diff
@@ -18,16 +18,51 @@ Do _NOT_ let certificates expire. If they expire, the environment becomes non-re
 . Once the host is in maintenance mode, click menu:Installation[Enroll Certificate].
 . Once enrollment is complete, click menu:Management[Activate].
 
-.Next steps
-You must also renew Engine certificates.
+[IMPORTANT]
+====
+You must also renew {engine-name} certificates.
+====
 
-. Run `engine-setup` from an SSH or console session on the {engine-appliance-name} system. The `engine-setup` script prompts you with some configuration questions.
+.Renewing certificates in a standalone {engine-name}
+. Log in as root via SSH or console session to the {engine-name} and run the following command:
 +
 ----
-# engine-setup
+# engine-setup --offline.
 ----
 +
+The `engine-setup` script prompts you with configuration questions.
+
 . Respond to the questions as appropriate or use an answers file.
++
+Be sure to answer the question about renewing certificates with "Y." This will renew the certificates without applying any updates.
++
+If you want to apply updates, follow the procedure in the Upgrade Guide.
+
+.Renewing certificates in a self-hosted engine
+
+. Log in as root via SSH or console session to the {engine-name} and a self-hosted engine hypervisor.
+
+. In the hypervisor, run the following command to place the environment in global maintenance mode:
++
+----
+# hosted-engine --set-maintenance –mode=global
+----
+. In the {engine-name}, run the following command to renew the certificates:
++
+----
+# engine-setup --offline.
+----
+
+. Respond to the questions as appropriate or use an answers file.
+Be sure to answer the question about renewing certificates with "Y." This will renew the certificates without applying any updates.
++
+If you want to apply updates, follow the procedure in the Upgrade Guide.
+
+. In the hypervisor, run the following command to remove the environment from global maintenance mode:
```

Comment:
hypervisor > host

The environment is not removed from a mode. The host is taken out of global maintenance mode. It looks like the term that is used is "[Activate](https://access.redhat.com/documentation/en-us/red_hat_virtualization/4.4/html-single/administration_guide/index#Moving_a_host_to_maintenance_mode) a host from global maintenance mode". That's not a turn of phrase I would have used but you might as well use it for the sake of consistency.

Reply:
I found a few different phrasings in our docs - "place" in global maintenance, "put" in global maintenance, 
but for turning it off, in most of the instances, I see "disable" global maintenance mode

Reply:
"Disable" sounds good to me.